runtime/oti/j2sever.h

```diff
@@ -70,12 +75,15 @@
 	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_V11
 #elif JAVA_SPEC_VERSION == 12
 	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_V12
+#elif JAVA_SPEC_VERSION == 13
+	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_V13
```

Comment:
J2SE_SHAPE_ doesn't seem to be used other than J2SE_SHAPE_RAW. Can we get rid of these constants?

Reply:
Sure.